content/python/concepts/inheritance/inheritance.md

```diff
@@ -68,28 +68,6 @@ Python here!
 
 **Note**: Inside the child class, the same number and type of arguments need to be passed into the parent class' method.
 
```

Comment:
```suggestion
```

Let's also delete this new line (which is probably causing the "lint" check to not pass).

Reply:
Got rid of it. Thanks.

Reply:
Hmm did we remove it? The GitHub checks still seems to not be passing. 🤔

Reply:
Done.